From 015c23060476e6808cf1103673361813823ed2de Mon Sep 17 00:00:00 2001
From: John Rinehart <johnrichardrinehart@gmail.com>
Date: Thu, 21 Aug 2025 06:23:05 -0700
Subject: [PATCH 13/14] fix: add of.h to rockchip_dvbm.c

According to claude, all of these are related to the missing header.
```
linux> ../drivers/video/rockchip/dvbm/rockchip_dvbm.c:731:35: error: implicit declaration of function 'of_match_ptr' [-Wimplicit-function-declaration]
linux>   731 |                 .of_match_table = of_match_ptr(rk_dvbm_dt_ids),
linux>       |                                   ^~~~~~~~~~~~
linux> ../drivers/video/rockchip/dvbm/rockchip_dvbm.c:731:35: error: initialization of 'const struct of_device_id *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
linux> ../drivers/video/rockchip/dvbm/rockchip_dvbm.c:731:35: note: (near initialization for 'rk_dvbm_driver.driver.of_match_table')
linux> ../drivers/video/rockchip/dvbm/rockchip_dvbm.c:731:35: error: initializer element is not constant
linux> ../drivers/video/rockchip/dvbm/rockchip_dvbm.c:731:35: note: (near initialization for 'rk_dvbm_driver.driver.of_match_table')
linux> make[6]: *** [../scripts/Makefile.build:287: drivers/video/rockchip/dvbm/rockchip_dvbm.o] Error 1
linux> make[5]: *** [../scripts/Makefile.build:555: drivers/video/rockchip/dvbm] Error 2
linux> make[4]: *** [../scripts/Makefile.build:555: drivers/video/rockchip] Error 2
```
---
 drivers/video/rockchip/dvbm/rockchip_dvbm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/rockchip/dvbm/rockchip_dvbm.c b/drivers/video/rockchip/dvbm/rockchip_dvbm.c
index 3e99ec4fda36b..205d6aa99fac7 100644
--- a/drivers/video/rockchip/dvbm/rockchip_dvbm.c
+++ b/drivers/video/rockchip/dvbm/rockchip_dvbm.c
@@ -9,6 +9,7 @@
 
 #include <linux/delay.h>
 #include <linux/module.h>
+#include <linux/of.h>
 #include <linux/of_platform.h>
 #include <linux/proc_fs.h>
 #include <linux/pm_runtime.h>
-- 
2.42.0

